src/koans/25_threading_macros.clj

```diff
@@ -0,0 +1,70 @@
+(ns koans.25-threading-macros
+  (:require [koan-engine.core :refer :all]))
+
+(def a-list
+  '(1 2 3 4 5))
+
+(def a-list-with-maps
+  '({:a 1} {:a 2} {:a 3}))
+
+(defn function-that-takes-a-map [m a b]
+  (do
+    (println (str "Other unused arguments: " a " " b))
+    (get m :a)))
+
+(defn function-that-takes-a-coll [a b coll]
+  (do
+    (println (str "Other unused arguments: " a " " b))
```

Comment:
Same question as above - I'd prefer to avoid printing here if possible, and I think it's only incidental to the point of these koans, so should be fine?

Reply:
Yeah, that makes sense. I basically wanted to show that if you have additional arguments then the position of the coll (and the scalar thing above) that the position of the coll/scalar matters.